aafd/file.te

```diff
@@ -1 +1 @@
-type p9fs, fs_type, mlstrustedobject;
+type p9fs, sdcard_type, fs_type, mlstrustedobject;
```

Comment:
Please move this line to vendor dir, since p9fs is declared in aafd, if aafd feature is not enabled, there will be a build error.

Reply:
done